TravelPoints-BackEnd/src/main/java/com/example/travelpointsbackend/controller/AuthenticationController.java

```diff
@@ -1,7 +1,41 @@
 package com.example.travelpointsbackend.controller;
 
-import org.springframework.stereotype.Controller;
+import com.example.travelpointsbackend.dto.UserDTO;
+import com.example.travelpointsbackend.model.auth.AuthenticationResponse;
+import com.example.travelpointsbackend.service.AuthenticationService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
 
-@Controller
+@CrossOrigin(origins = "http://localhost:4200")
+@RestController
+@RequestMapping("/auth")
+@RequiredArgsConstructor
+@Slf4j
 public class AuthenticationController {
+    @Autowired
+    private AuthenticationService authenticationService;
+
+//    @PostMapping("/register")
+//    public ResponseEntity<AuthenticationResponse> registerTourist(@RequestBody UserDTO userDTO){
+//        return ResponseEntity.ok(authenticationService.registerTourist(userDTO));
+//    }
+//
+//    @PostMapping("/registerAdministrator")
+//    public ResponseEntity<AuthenticationResponse> registerAdministrator(@RequestBody UserDTO userDTO){
+//        return ResponseEntity.ok(authenticationService.registerAdministrator(userDTO));
+//    }
+
+    @PostMapping("/login")
+    public ResponseEntity<AuthenticationResponse> login(@RequestBody UserDTO userDTO ){
+        return ResponseEntity.ok(authenticationService.login(userDTO));
+    }
+
+//    @PutMapping("/logout/{userId}")
+//    public ResponseEntity<String> logout(@PathVariable Long userId){
+//        authenticationService.logOut(userId);
+//        return ResponseEntity.ok("Logged!");
+//    }
```

Comment:
S: why is all of this commented?

Reply:
Register and logout are part of other tasks, but I implemented them together with security to be able to test it.